test/helpers/team_member_helper_test.rb

```diff
@@ -0,0 +1,7 @@
+require 'test_helper'
+
+class TeamMemberHelperTest < ActionView::TestCase
```

Comment:
What's the purpose of this test? Possibly just me not knowing Rails

Reply:
It's a test I should have written but never did...